lib/assetmanager.js

```diff
@@ -198,6 +198,9 @@ exports = module.exports = {
     } else {
       options.populate = _.extend(pop, options.populate);
     }
+
+    // Extend the search by _isDeleted to not return the soft deleted assets
+    search = _.extend(search, {_isDeleted: false});
```

Comment:
can we check search first to check if `isDeleted` is already set, and only default to false if not? That way, the api can still be used to fetch deleted assets if, for example, we want to add a gui to inspect assets that have been deleted.